hkmc2/shared/src/main/scala/hkmc2/codegen/js/JSBuilder.scala

```diff
@@ -329,7 +329,12 @@ class JSBuilder extends CodeBuilder:
   def body(t: Block)(using Raise, Scope): Document = scope.nest givenIn:
     block(t)
   
-  
+  def handleFunction(ps: List[semantics.Param], b: Block)(using Raise, Scope): (Document, Document) =
```

Comment:
Let's rename this to
```suggestion
  def setupFunction(params: List[semantics.Param], body: Block)(using Raise, Scope): (Document, Document) =
```
As "handle" is a bit too generic.